homeassistant/components/lutron_caseta/fan.py

```diff
@@ -84,6 +84,6 @@ async def async_set_percentage(self, percentage: int) -> None:
         await self._smartbridge.set_fan(self.device_id, named_speed)
 
     @property
-    def is_on(self):
+    def is_on(self) -> bool:
         """Return true if device is on."""
-        return self.percentage and self.percentage > 0
+        return self.percentage is not None and self.percentage > 0
```

Comment:
I've had to add this check to pass `mypy`.
Another option might be to adjust the type hint to be nullable and do it like this:
```suggestion
        if self.percentage is None:
            return None
        return self.percentage > 0
```

Reply:
This can be replaced by:

```suggestion
        return bool(self.percentage)
```